src/main/resources/assets/astralsorcery/lang/en_us.json

```diff
@@ -1132,10 +1133,10 @@
   "perk.astralsorcery.special.connector.armara.name": "Chakra of Tenacity",
   "perk.astralsorcery.special.connector.vicio.name": "Chakra of Dexterity",
   "perk.astralsorcery.special.connector.aevitas.name": "Chakra of Creation",
-  "perk.astralsorcery.special.connector.key.evorsio.name": "Epiphany: Transience",
+  "perk.astralsorcery.special.connector.key.evorsio.name": "Epiphany: Destruction",
   "perk.astralsorcery.special.connector.key.evorsio.desc.1": "Other 'Epiphany' nodes may be unlocked without",
   "perk.astralsorcery.special.connector.key.evorsio.desc.2": "requiring a connection to the existing perks",
-  "perk.astralsorcery.special.connector.key.discidia.name": "Epiphany: Destruction",
+  "perk.astralsorcery.special.connector.key.discidia.name": "Epiphany: Blight",
```

Comment:
Destruction is Evorsio's primary aspect, not Discidia's. Discidia is there to cause harm to others, not blow things up.
Unless I am missing something in the bigger picture

Return to me if you still want this change undone

Reply:
For what it's worth, I noticed this in my earlier PR (just didn't change it since it felt outside of the scope of my changes) and I agree with @yagoplx . 

Evorsio's subtitle is literally _Destruction_ in the tome, so it doesn't make a lot of sense that it's used here for Discidia.

Reply:
The idea behind the epiphany names was to use a combination between the 2 roots they're between.
That wasn't perfectly done, sure, but these names don't capture that either. Also, rename the Perks around it as well when renaming the epiphany node.

Reply:
addressed